### PR TITLE
refactor: clean up client HTML; fix tiny ts errors;

### PR DESCRIPTION
**Pull request by lmssiehdev**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "build": "vite build",
+    "build": "tsc && vite build",
```
Comment (leia-uwu) [inline client/package.json:4]: why this

**Review by leia-uwu — CHANGES_REQUESTED** [review]
i would like to keep PWA metadata so people can install it from the browser on phones

File: client/package.json (hunk 1 above)
Comment (lmssiehdev) [reply client/package.json:4]: Vite does not perform type checking

Comment (leia-uwu) [reply client/package.json:4]: hmm change it to `tsc --noEmit` so it doesn't output the files 
but maybe we should just use this https://github.com/fi3ework/vite-plugin-checker to get TS errors in the browser on dev server

**Comment by leia-uwu** [comment]
btw is the pwd generator dep really necessary? it requires libvips to be installed on linux otherwise `pnpm install` will fail

**Comment by leia-uwu** [comment]
tbh it seems more like a tool you would install globally on your system to use the cli, i will remove it from the dependency for now but keep the package.json script

**Comment by leia-uwu** [comment]
:stuck_out_tongue: i don't have push permission to your repo so i'm going to merge and do my changes after

**Pull request merged**